### DISCUSSION TITLE
Where to hook into time that shows up in the terminal

### DISCUSSION THREAD

**Discussion by sturentzler** [Comfy-Org/ComfyUI#1846] [Q&A]
Hi - when running Comfyui, the terminal shows the progress along with the estimated time, this part: 00:26<00:00.  Which script is outputting progress to the terminal and showing in particular the time part? Thank you!

100%|███████████████████████████████████████████████████| 20/20 **[00:26<00:00**,  1.34s/it]

**Comment by elthariel** [comment]
It comes from a library called `tqdm`. It is called by the various samplers directly. 
Search the codebase for `tqdm` or `trange`. You sadly can't really hook into it
